automationspn/script.ps1

```diff
@@ -0,0 +1,29 @@
+#Create a local function project
+#Choose folder name
+$foldername = Read-Host -Prompt "Enter a name for your folder" 
+
+#create local function project in powershell
+func init $foldername --powershell
+
+#navigate into the project folder
+cd $foldername
+
+#create unique name for HTTP Trigger function
+$functionname = Read-Host -Prompt "Name your function"
+#add function to the project
+func new --name $functionname --template "HTTP trigger" --authlevel "anonymous" 
```

Comment:
Same here - this part creates a function (and generates code) on your local machine. This is a separate task. You should remove that